clang/test/Driver/Inputs/test1.ll

```diff
@@ -0,0 +1,28 @@
+; ModuleID = 'test1.cpp'
+source_filename = "test1.cpp"
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64-G1"
+target triple = "spirv64"
+
+; Function Attrs: mustprogress noinline
+define spir_func noundef i32 @_Z9foo_func1ii(i32 noundef %a, i32 noundef %b) local_unnamed_addr #0 {
```

Comment:
Please, use one notation for function prefixes and file names.
Either rename `test1.ll` to `foo.ll` and `test2.ll` to `bar.ll` or use `test1_` and `test2_` function name prefixes.